test/test_rospy/test/unit/test_rospy_core.py

```diff
@@ -39,9 +39,14 @@
 import unittest
 import time
 import random
+from StringIO import StringIO
```

Comment:
This doesn't work with Python 3. Please use the same conditional import as in the other files.